src/ui/layout/BrowseBar.vue

```diff
@@ -142,7 +142,13 @@ import NotebookMenuSwitcher from '@/plugins/notebook/components/NotebookMenuSwit
 import IndependentTimeConductor from '@/plugins/timeConductor/independent/IndependentTimeConductor.vue';
 import tooltipHelpers from '../../api/tooltips/tooltipMixins';
 
-const SupportedViewTypes = ['plot-stacked', 'plot-overlay', 'bar-graph.view', 'time-strip.view'];
+const SupportedViewTypes = [
```

Comment:
Do we need to add this to ObjectFrame.vue too? I think that is when it's in a display layout...

Reply:
Yes!